src/components/schema-ui/schema-menu-attribute.js

```diff
@@ -77,7 +83,7 @@ const SchemaMenuAttributeValue = ({ name, value, forPath, level }) => {
           {level === 0 && <AttributeFocusToggle path={[...forPath, name]} />}
         </span>
         <span className="link__text link__text--machine">{name}</span>
-        {type !== 'object' && <span className="link__text_type link__text--machine">{type}</span>}
+        {<span className="link__text_type link__text--machine">{type}</span>}
```

Comment:
nit: you can remove the `{}` since there's no logic now

Reply:
Oh, good call.